web/debugerror.py

```diff
@@ -355,7 +355,7 @@ def emailerrors_internal():
             "your buggy site <%s>" % from_address,
             "the bugfixer <%s>" % to_address,
             "bug: %(error_name)s: %(error_value)s (%(path)s)" % locals(),
-            message,
+            message.encode('utf-8'),
```

Comment:
We run https://pypi.org/project/black
```suggestion
            message.encode("utf-8"),
```